spec.bs

```diff
@@ -234,6 +234,34 @@ Browsing contexts -- that are in the background and have their documents in <a h
 When a [=top-level browsing context=] (tab in the browser) is [=/discarded=] due to resource pressure (or unexpected events eg. process crash), and later the user revisits the tab in the browser, then the {{Document}}'s [=Document/discarded=] boolean will be true due to [[#html-initialize-doc]].
 
 
+Feature Policies {#feature-policies}
+=====================
+To improve CPU utilization of sub-frames that are not critical to a page's usability two feature policies are defined:
+ * execution-while-not-rendered, which has a <a href="https://wicg.github.io/feature-policy/#default-allowlist">default allowlist</a> of '*'
+ * execution-while-out-of-viewport, which has a <a href="https://wicg.github.io/feature-policy/#default-allowlist">default allowlist</a> of '*'
+
+<dfn export lt="execution-while-not-rendered">execution-while-not-rendered</dfn> policy controls whether [=task source/freezable=]
+tasks should execute for frames that are not part of rendering. eg. `display: none`.
+
+<dfn export lt="execution-while-out-of-viewport">execution-while-out-of-viewport</dfn> policy controls whether [=task source/freezable=]
```

Comment:
lt="" should not be necessary if the text is the same inside the tag

Reply:
Done